app/models/card.rb

```diff
@@ -16,4 +20,21 @@ def check_duplication
   def set_default_date
     self.review_date = Date.today + 3.days
   end
+
+  def check_translations(submitted_text)
+    puts prepare_tex(self.original_text)
```

Comment:
Redundant `self` detected.<br>Do not write to stdout. Use Rails' logger if you want to log.